website/source/docs/agent/options.html.md

```diff
@@ -1147,6 +1147,11 @@ default will automatically work with some tooling.
     * <a name="dns_cache_max_age"></a><a href="#dns_cache_max_age">`cache_max_age`</a> - When [use_cache](#dns_use_cache) is enabled, the agent
       will attempt to re-fetch the result from the servers if the cached value is older than this duration. See: [agent caching](/api/features/caching.html).
 
+    * <a name="prefer_namespace"></a><a href="#dns_prefer_namespace">`prefer_namespace`</a> - ** Enterprise Only ** When
```

Comment:
This link sends me to the top of the configuration page, not back to the same menu option. Also, "Enterprise Only" has spaces between it and the asterisks to the bold doesn't render.

Reply:
Fixing